pages/musicalbum/_itemId/index.vue

```diff
@@ -0,0 +1,190 @@
+                :sm="twoColsInfoColumn.lSm"
+                :class="twoColsInfoColumn.lClass"
+              >
+                <label class="text--secondary">{{ $t('genres') }}</label>
+              </v-col>
+              <v-col
+                class="px-0"
+                :cols="twoColsInfoColumn.rCols"
+                :sm="twoColsInfoColumn.rSm"
+              >
+                <v-slide-group>
+                  <v-slide-item
+                    v-for="(genre, index) in item.GenreItems"
+                    :key="`genre-${genre.Id}`"
+                  >
+                    <v-chip
+                      small
+                      link
+                      :class="{ 'ml-2': index > 0 }"
+                      nuxt
+                      :to="`/genre/${genre.Id}?type=${item.Type}`"
+                    >
+                      {{ genre.Name }}
+                    </v-chip>
+                  </v-slide-item>
+                </v-slide-group>
+              </v-col>
+            </v-row>
+          </v-col>
+        </v-col>
+      </v-row>
+      <v-row>
+        <v-col cols="12">
+          <track-list
+            v-if="item.Type === 'MusicAlbum'"
+            :item="item"
+          ></track-list>
+        </v-col>
+      </v-row>
+    </template>
+    <template #right>
+      <related-items :item="item" vertical />
+    </template>
+  </item-cols>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { mapActions } from 'vuex';
+import { BaseItemDto, ImageType } from '@jellyfin/client-axios';
+import imageHelper from '~/mixins/imageHelper';
+import formsHelper from '~/mixins/formsHelper';
+import itemHelper from '~/mixins/itemHelper';
+
+interface TwoColsInfoColumn {
+  lCols: number;
+  lSm: number;
+  rCols: number;
+  rSm: number;
+  lClass: { [key: string]: boolean };
+}
+
+export default Vue.extend({
+  mixins: [imageHelper, formsHelper, itemHelper],
+  async asyncData({ params, $api, $auth }) {
+    const item = (
+      await $api.userLibrary.getItem({
+        userId: $auth.user?.Id,
+        itemId: params.itemId
+      })
+    ).data;
+
+    return {
+      item
+    };
+  },
+  data() {
+    return {
+      item: {} as BaseItemDto
+    };
+  },
+  head() {
+    return {
+      title: this.$store.state.page.title
+    };
+  },
+  computed: {
+    twoColsInfoColumn: {
+      get(): TwoColsInfoColumn {
+        return {
+          lCols: 12,
+          lSm: 2,
+          lClass: {
+            'mt-3': !this.$vuetify.breakpoint.smAndUp,
+            'py-0': !this.$vuetify.breakpoint.smAndUp,
+            'px-0': true,
+            'text-truncate': true
+          },
+          rCols: 12,
+          rSm: 10
+        };
+      }
+    }
+  },
+  beforeMount() {
+    if (this.item) {
+      this.setPageTitle({ title: this.item.Name });
+      this.setAppBarOpacity({ opaqueAppBar: false });
+      const hash = this.getBlurhash(this.item, ImageType.Backdrop);
+      this.setBackdrop({ hash });
```

Comment:
Hmm, what about setting this in an immediate watcher, so changes of the metadata while in the page are applied (and you also don't need beforeMount anymore)

Reply:
I moved it to a deep, immediate watcher on all the pages. setAppBarOpacity was moved to `created`, so that it's earlier in the lifecycle and happens server-side for the SSR version.